src/validation.cpp

```diff
@@ -1131,7 +1131,13 @@ CAmount GetBlockSubsidyInner(int nPrevBits, int nPrevHeight, const Consensus::Pa
         dDiff = ConvertBitsToDouble(nPrevBits);
     }
 
-    if (nPrevHeight < 5465) {
+    if (fV20Active) {
+        // Starting from V20 Activation, subsidybase should be stable.
```

Comment:
linking here for visibility https://github.com/dashpay/dash/commit/63ef4935b5d09e83152cb29dc935ad568fdb4600#r130063059